app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge = @project.challenge
```

Comment:
It's totally fine to do `@project.challenge` in a view so your @challenge instance variable on line 4 is not strictly necessary